shared/naturalcrit/markdown.js

```diff
@@ -206,6 +206,38 @@ const mustacheInjectBlock = {
 	}
 };
 
+const superSubScripts = {
+	name  : 'superSubScripts',
+	level : 'inline',
+	start(src) { return src.match(/.*\^\^(.+)\^\^/)?.index; },  // Hint to Marked.js to stop and check for a match
+	tokenizer(src, tokens) {
+		const superRegex = /\^\^(.+)\^\^/y;
```

Comment:
The Marked.js package we use recommends these should start with an unescaped `^` at the beginning and use the `m` regex tag to make sure these only match at the start of the current string. It's possible for the markdown parser to mistakenly consume extra characters in certain cases if it finds a match later on in the string behind other tokens.

Reply:
I made that change on the tokenizer regexes but when made on the start match it fails to match.

Reply:
Yeah this only applies to the regex inside the tokenizer.